scripts/bump_monorepo_packages.js

```diff
@@ -0,0 +1,118 @@
+#!/usr/local/bin/node
+const fs = require( "fs" );
+const execSync = require( "child_process" ).execSync;
+
+// Non-@yoast packages.
+const legacyYoastPackages = [ "yoast-components", "yoastseo", "eslint-config-yoast" ];
+
+// Console colors.
+const bright = "\x1b[1m";
```

Comment:
I would prefix this with `color` and maybe it's nicer to uppercase these constants.